manual/manual/refman/exten.etex

```diff
@@ -1635,43 +1635,63 @@ Some attributes are understood by the type-checker:
    default is currently "boxed" but it may change in the future.
 \end{itemize}
 
-\begin{verbatim}
-  [@@deprecated "Please use module 'Y' instead."]
+\begin{caml_example*}{verbatim}
+module X = struct
+  [@@@warning "+9"]  (* locally enable warning 9 in this structure *)
+  [@@@ellipsis]
+end
+[@@deprecated "Please use module 'Y' instead."]
 
-let x = begin[@warning "+9"] ... end in ....
+let x = begin[@warning "+9"] ()[@ellipsis] end
```

Comment:
There is a small technical issue here: `begin[@warning "+9"] ()[@ellipsis] end` is syntactically equivalent to `()[@warning "+9"][@ellipsis]`, thus everything get elided. A possible fix is to replace `()[@ellipsis]` by `[()[@ellipsis]]` .